doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+
+\* - different than PTS defaults
+
+\^ - field not available on PTS
+
+M - mandatory
+
+O - optional
+
+
+Major Profile Version (X.Y)
+===========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_0_1	True		Mesh v1.0 (M)
+==============	==============	=======================================
+
+
+Minor Profile Version (X.Y.Z)
+=============================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_0A_1	True		Erratum 10395 (C.1)
+TSPC_MESH_0A_2	True		Mesh v1.0.1 (C.2)
+==============	==============	=======================================
+
+
+Bluetooth Core Specification v4.0 and Later
+===========================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_1_1	True		Core Specification 4.0 or later version (M)
+==============	==============	=======================================
+
+
+Roles
+=====
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_2_1	True		Node (C.1)
+TSPC_MESH_2_2	False (*)	Provisioner (C.1)
+==============	==============	=======================================
+
+
+Node Capabilities - Bearers
+===========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_3_1	True		Advertising Bearer (C.1)
+TSPC_MESH_3_2	True		GATT Bearer (C.1)
+==============	==============	=======================================
+
+
+Provisioning
+============
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_4_1	True		PB-ADV (C.1)
+TSPC_MESH_4_2	True		PB-GATT (C.2)
+TSPC_MESH_4_3	True		Device UUID (C.3)
+TSPC_MESH_4_4	True		Sending Unprovisioned Device Beacon (C.4)
+TSPC_MESH_4_5	True		Generic Provisioning Layer (C.3)
+TSPC_MESH_4_6	True		Provisioning Protocol (C.3)
+TSPC_MESH_4_7	False (*)	Provisioning: Public Key OOB (C.5)
+TSPC_MESH_4_8	True		Provisioning: Public Key No OOB (C.5)
+TSPC_MESH_4_9	True		Provisioning: Authentication Output OOB (C.6)
+TSPC_MESH_4_10	False (*)	Provisioning: Authentication Input OOB (C.6)
+TSPC_MESH_4_11	False (*)	Provisioning: Authentication Static OOB (C.6)
+TSPC_MESH_4_12	True		Provisioning: Authentication No OOB (C.3)
+==============	==============	=======================================
+
+
+Node Capabilites - Network Layer
```

Comment:
``Capabilities`` misspelling is still here (search and replace ALL misspelled ``Capabilites``)

Reply:
Thanks.